com.gitlab.jakobdev.jdnbtexplorer.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.gitlab.jakobdev.jdnbtexplorer.metainfo</id>
+  <name>jdNBTExplorer</name>
+  <summary>A Editor for Minecraft NBT files</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <content_rating type="oars-1.0"/>
+  <description>
+    <p>jdNBTExplorer is a NBT Editor for MInecraft that has a QT6 GUI.</p>
+  </description>
+  <launchable type="desktop-id">com.gitlab.jakobdev.jdnbtexplorer.metainfo.desktop</launchable>
+  <url type="homepage">https://gitlab.com/JakobDev/jdNBTExplorer</url>
+  <url type="bugtracker">https://gitlab.com/JakobDev/jdNBTExplorer/-/issues</url>
+  <screenshots>
+    <screenshot type="default">https://gitlab.com/JakobDev/jdNBTExplorer/-/raw/master/Screenshots/MainWindow.png</screenshot>
+    <screenshot>https://gitlab.com/JakobDev/jdNBTExplorer/-/raw/master/Screenshots/EditWindow.png</screenshot>
```

Comment:
you should reference a tag or revision. Because with this if tomorrow `master` move or remove these files the build will break.